share/spice/movie/movie.js

```diff
@@ -12,7 +12,119 @@
         return DDG.get_asset_path('in_theaters', critics_rating + ((DDG.is3x || DDG.is2x) ? '.retina.png' : '.png'));
     }
 
-    env.ddg_spice_movie = function(api_result) {
+    function normalizeMovie(item) {
+        if (item.alternate_ids && item.alternate_ids.imdb) {
+            return {
+                rating: Math.max(item.ratings.critics_score / 20, 0),
+                icon_image: getImage(item.ratings.critics_rating),
+                abstract: Handlebars.helpers.ellipsis(item.synopsis || item.critics_consensus, 200),
+                heading: item.title,
+                fallback_image: item.posters.detailed,
+                image: null,
+                url: item.links.alternate,
+                is_retina: (DDG.is3x || DDG.is2x) ? "is_retina" : "no_retina"
+            };
+        }
+    }
+
+    function normalizeCast(item) {
+        return {
+            title: item.name,
+            altSubtitle: item.characters && item.characters.join(", "),
```

Comment:
I think we should preface this with "as" so it's clear this is the character name?

>**Christian Bale**
>as Bruce Wayne, Batman